team/ben-smith.yaml

```
@@ -0,0 +1,21 @@
+# Replace CAPITALIZED placeholders with your info
+- title: FIRST LAST
```

```suggestion
- title: Ben Smith
```
Unless you've changed your name... ; )